### PR TITLE
Exercises five to ten

### PR DESCRIPTION
**Pull request by Liodas**
Architecture rework to have the exercises separated from the 'main' code.

Domain.fs and Operations.fs have been added for one of the exercises using multi-files work. (Is there a naming convention for files ? Always singular or plural ?)

Exercises 1 to 5 stayed the same, they just have been moved to the newly created folder 'Scripts'. Exercises 6 to 10 are new.

### CODE REVIEW

File: MyFirstSharpApp/scripts/Exercise10fsx.fsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿// The goal of this exercise is to
+
+open System.Net
+
+let postUsername (client: WebClient) name =
+    client.UploadString ("https://httpbin.org/post", "fname=" + name);
+
+let postAge (client: WebClient) age =
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise10fsx.fsx:8]: Same remark about the position of the web client.

File: MyFirstSharpApp/scripts/Exercise10fsx.fsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    client.UploadString ("https://httpbin.org/post", "age=" + age)
+
+let postInformations name age =
+    // Can't use 'use' keyword outside of a function
+    use client = new WebClient ()
+    // <- used to modify value after init
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise10fsx.fsx:14]: Yes and no :)

That's the assignment operator (like `=` in C#). Most of the time this is used to set a property/

File: MyFirstSharpApp/scripts/Exercise10fsx.fsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+open System.Net
+
+let postUsername (client: WebClient) name =
+    client.UploadString ("https://httpbin.org/post", "fname=" + name);
+
+let postAge (client: WebClient) age =
+    client.UploadString ("https://httpbin.org/post", "age=" + age)
+
+let postInformations name age =
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise10fsx.fsx:11]: FYI `information` is left un-modified on the plural form.

File: MyFirstSharpApp/scripts/Exercise3.fsx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+let createForm uri =
+    let fsharpOrg =
+        use webClient = new WebClient()
+        webClient.DownloadString(Uri uri)
+
+    use browser =
+        new WebBrowser
+            (ScriptErrorsSuppressed = true
+            , Dock = DockStyle.Fill
+            , DocumentText = fsharpOrg)
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise3.fsx:16]: Note the space after and before parenthesis.

```suggestion
            ( ScriptErrorsSuppressed = true
            , Dock = DockStyle.Fill
            , DocumentText = fsharpOrg )
```

File: MyFirstSharpApp/scripts/Exercise5.fsx [hunk 5]
```diff
@@ -0,0 +1,13 @@
+﻿//The goal of this exercise is to learn about tuples and immutable variables, even if, in this case, it is simple to implement without a tuple
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise5.fsx:1]:
```suggestion
﻿// The goal of this exercise is to learn about tuples and immutable variables, even if, in this case, it is simple to implement without a tuple
```

File: MyFirstSharpApp/scripts/Exercise6.fsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+﻿//The goal of this exercise is to :
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise6.fsx:1]:
```suggestion
﻿// The goal of this exercise is to :
```

File: MyFirstSharpApp/scripts/Exercise7.fsx [hunk 7]
```diff
@@ -0,0 +1,17 @@
+﻿// The goal of this exercise is simply to learn how to create records and init them.
+
+type Car =
+    {
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise7.fsx:4]: Please apply styling pattern suggested earlier.

File: MyFirstSharpApp/scripts/Exercise7.fsx [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+        Manufacturer : string
+        EngineSize : string
+        NbDoors : int
+        NbWheels : int
+    }
+
+let _car =
+    {
```
Comment (Evangelink) [inline MyFirstSharpApp/scripts/Exercise7.fsx:12]: Please apply styling pattern suggested earlier.

File: MyFirstSharpApp/scripts/Exercise10fsx.fsx (hunk 2 above)
Comment (Liodas) [reply MyFirstSharpApp/scripts/Exercise10fsx.fsx]: image

I was talking about the FS0524 warning that I get if I declare it outsite of a function, I don't know if it's what you were referring too ? :)

File: MyFirstSharpApp/Operations.fs [hunk 9]
```diff
@@ -0,0 +1,8 @@
+﻿module Operations
+open Domain
+
+let getInitials customer =
+    customer.FirstName.[0], customer.LastName.[0]
+
+let isOlderThan age customer =
+    customer.Age > age
```
[the remark this answers is not in the document]
Comment (Liodas) [reply MyFirstSharpApp/Operations.fs:8]: Alright, I'll keep that in mind.

File: MyFirstSharpApp/Domain.fs [hunk 10]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Domain
+
+type Customer =
+    { FirstName: string
+      LastName : string
+      Age : int }
```
Comment (Evangelink) [inline MyFirstSharpApp/Domain.fs:6]: You forgot to apply the no space before colon change here.

File: MyFirstSharpApp/scripts/Exercise10fsx.fsx (hunk 3 above)
Comment (Evangelink) [reply MyFirstSharpApp/scripts/Exercise10fsx.fsx:15]: Ping

**Comment by Evangelink** [comment]
> Domain.fs and Operations.fs have been added for one of the exercises using multi-files work. (Is there a naming convention for files ? Always singular or plural ?)

Usually singular

**Pull request merged**